packages/gateway/package.json

```diff
@@ -3,12 +3,10 @@
     "version": "1.0.0",
     "main": "index.js",
     "scripts": {
+        "rover": "rover supergraph compose --config ./supergraph-config.yaml > supergraph.graphql",
```

Comment:
This is the bit that I spent 2+ hours banging my head against... and what you have done is the same. So I am super confused, but glad it's working 🥴